config/sidekiq.yml

```diff
@@ -9,6 +9,8 @@
   - events
   - scheduled
   - exports
+  - user_report
```

Comment:
@Quentinchampenois @AyakorK the queue `user_report` is defined twice

I'm fixing it with another PR

![image](https://github.com/user-attachments/assets/5b9b2eb0-ad7e-4822-8577-d6d93a58f6fc)